app/components/layout/navbar/page.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+
+import styles from "@/components/layout/navbar/navbar.module.css";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+
+const NAV_ITEMS = [
+  { path: "/", label: "ホーム" },
+  { path: "/timeline", label: "タイムライン" },
+  { path: "/trend", label: "トレンド" },
+  { path: "/question-feed", label: "質問" },
+  { path: "/official-events", label: "公式イベント" },
+  { path: "/official-columns", label: "公式コラム" },
+  { path: "/organizations", label: "Organization" },
+];
+
+export default function Navbar() {
+  const pathName = usePathname();
+
+  const getNavItemClass = (path: string) =>
+    `${styles.navbarItem} ${pathName === path ? styles.active : ""}`.trim();
```

Comment:
```suggestion
  const addedClassName = pathName === path ? styles.active : "";
  const getNavItemClass = (path: string) =>
    `${styles.navbarItem} ${addedClassName}`.trim();
```

今のコードでももちろん良いですが、文字列結合しつつ、三項演算子をセットするのはやや可読性が損なわれると感じます。

`addedClassName（追加のクラス）` を明示的に書くことで、あぁ差分はこれなんだな、がわかりやすく伝えられると思います